src/client.js

```diff
@@ -541,6 +543,61 @@ class Client {
          * @type {Campaign.Application}
          */
         this.application = null;
+
+        // temporary start in 2000
+        this.lastTime = "2000-01-01T00:00:00.000";
+        this.buildNumber = "0"; // temporary build number start
```

Comment:
I don't understand this. It looks like a hack. If not, can you add a very detailed comment explaining the intent and the logic. Also why a string and not a number?

Reply:
I used a string because buildNumber is stored as a string in localstorage.